english/macgyver/toss-lighter/toss-lighter.md

```diff
@@ -1,2 +1,10 @@
 When the lighter contacts the marshmallow, a flame ignites
 the marshmallow wall instantly. The marshmallow burns away revealing a hidden room.
+
+The room smells "musky" but it seems like your only option at this point.
+
+Do you:
+
+[Stand there like a goon](Stand/Stand_like_goon.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.